test/wallet.js

```diff
@@ -313,7 +328,7 @@ function signatureTest({ testLabel, method, privateKey, addressHex, message, sig
 
   var payload = {
     method: method,
-    params: [addressHex, message],
+    params: [message, addressHex],
```

Comment:
this seems out of bounds -- is this the correct param ordering for all signing methods?

Reply:
Yes, [message, address] is the standard for the signing methods. We support the other order when concretely detected to support a time when it was less standard.

Reply:
Timing for this line and warning in [HookedWallet](https://github.com/MetaMask/provider-engine/blob/master/subproviders/hooked-wallet.js#L139) suggested this is the right way.

Reply:
ok but what about `eth_sign`?
I would expect this to break `eth_sign` tests (tho it doesnt seem to)

Reply:
I see, we dont use this helper for `eth_sign` tests :+1: